autorel/changelog_generator/main/changelog_structures.py

```diff
@@ -11,6 +11,7 @@ class Contributor(object):
     def __init__(self, name, email, url):
         self._name = name
         self._email = email
+        self._url = url
```

Comment:
Sorry, I really wanted to review your changes, but we had some urgent issues related with syslog-ng.
I'll definitely continue the review tomorrow.